sample-compose/app/build.gradle.kts

```diff
@@ -101,6 +111,7 @@ android {
                 }
             }
         }
+        animationsDisabled = true
```

Comment:
@luongvo Did we add this in order to "speed up" the tests? Just wondering if it would be useful to keep animations enabled, then we might be able to discover some bugs (caused by animation)  💭

Reply:
@Wadeewee @Tuubz Alright, I think we can disable this with a comment to remind us if we need to turn the animation off later 👌

Reply:
Disabled with comment https://github.com/nimblehq/android-templates/pull/384/commits/bda2a6a742ae459b455588e700ca4d387716ec29